external/fv3fit/fv3fit/reservoir/train.py

```diff
@@ -37,6 +36,44 @@ def _add_input_noise(arr: np.ndarray, stddev: float) -> np.ndarray:
     return arr + np.random.normal(loc=0, scale=stddev, size=arr.shape)
 
 
+def _get_transformers(
+    sample_batch: Mapping[str, tf.Tensor], hyperparameters: ReservoirTrainingConfig
+) -> TransformerGroup:
+    # Load transformers with specified paths
+    transformers = {}
+    for variable_group in ["input", "output", "hybrid"]:
+        path = getattr(hyperparameters.transformers, variable_group, None)
+        if path is not None:
+            transformers[variable_group] = cast(Transformer, fv3fit.load(path))
+
+    # If input transformer not specified, always create a standard norm transform
+    if "input" not in transformers:
+        transformers["input"] = get_standard_normalizing_transformer(
+            hyperparameters.input_variables, sample_batch
+        )
+
+    # If output transformer not specified and output_variables != input_variables,
+    # create a separate standard norm transform
+    if "output" not in transformers and (
+        hyperparameters.output_variables != hyperparameters.input_variables
+    ):
+        transformers["output"] = get_standard_normalizing_transformer(
+            hyperparameters.output_variables, sample_batch
+        )
+    else:
+        transformers["output"] = transformers["input"]
+
+    # If hybrid variables transformer not specified, and hybrid variables are defined,
+    # create a separate standard norm transform
+    if "hybrid" not in transformers and hyperparameters.hybrid_variables is not None:
```

Comment:
Same situation as `output` transformer above.  Move the `hybrid_variables` check into the `not in transformers` block.